### ISSUE TITLE
Bias on Elemental Damage going through physical invulnerabilities

### ISSUE THREAD

**Issue by theloras** [elysium-project/server#725]
https://cdn.discordapp.com/attachments/253215035613118467/313579359644418049/elemental_damage_through_invuln.jpg

Rogue poisons go through BoP, but SoR can't proc when a mage pops a limited invulnerability potion?

And no, he wasn't Iceblocked either - he popped an invulnerability pot

**Comment by killerduki** [comment]
Theloras , i think it's little misunderstanding , apparently SoR/SoC can/should proc only on "successful" hit , not on swings that got "immune" .

Try to pick weapon that has +frost/nature damage on their base attack, you will see then SoR/SoC working with that one.

**Comment by Imbaslap** [comment]
do you have a screenshot showing chance on hit effects proccing and going through a BOP?
i know what the bug is, and its something else entirely.

**Comment by theloras** [comment]
SoR and SoC can still proc if the white attack doesn't land - I originally thought that they shouldn't either - but on every private server, Rogue poisons still are able to go through BoP.

Whether that is blizzlike, I am not 100% sure but the fact that there is a double standard here with elemental damage attacks should be addressed.

**Comment by Phoosy** [comment]
If it is indeed Blizzlike then there may be nothing to address, as the implementation on core emulation would be intentional at that point - I'm not saying it's not an oversight, but if it operated that way back on live then it was a rather crappy oversight amongst many that vanilla servers have to live with.

**Comment by theloras** [comment]
It just strikes me as incredibly odd that if it is Blizzlike, that every other spell school's elemental damage is able to proc against a physical damage immune target.
@GuybrushGit - what are your thoughts on this?

**Comment by killerduki** [comment]
@theloras  the difference between SoR and other damage is that Rogues to proc Poisons they don't need to land successful melee swing (fix me if i am incorrect) , while SoR does require , unless you have weapon with +elemental of any kind of school including holy.

**Comment by theloras** [comment]
that's just it Duki - originally I also thought that for SoR or SoC to proc, the white attack needed to land - but someone showed me a video of it not being the case

I can't remember if it was on Nost or Feenix - trying to find the clip

**Comment by Phoosy** [comment]
Evidence from Nost or Feenix is not admissable - only vanilla WoW content is a valid representation.

**Comment by theloras** [comment]
no it was a retail vanilla video - i just wasnt sure if the person showed it to me while i was playing on nost or feenix - sorry for the confusion

**Comment by theloras** [comment]
Duki and I did some testing just now with the low level Reforged Blade of Heroes as it has a +5-10 Fire damage attack on each white attack.

Duki used Blessing of Protection and SoR proc'd on each of the elemental damage attacks:

https://cdn.discordapp.com/attachments/253446111753273354/313914217214377994/SoR_proc_elemental.jpg

**Comment by theloras** [comment]
Found it - not video evidence but a thread discussion - there may be video evidence as well.

It was back on Feenix where we argued whether the white attack needed to land for a Seal to proc - I had originally thought that SoC couldn't be Block/Dodge/Parry and I also thought that the white attack had to land:

http://www.classicwow.com/forum/topic/61649-seal-of-command-fixes/?do=findComment&comment=486341

**Comment by theloras** [comment]
My original though was:

1) Did the Auto-Attack white hit land or did it Miss, Dodge, Block or Parry - if yes then,

2) Did Seal of Command proc off of the white attack - if yes then,

3) Did the SoC yellow hit land or did it Miss, Dodge, Block or Parry?

When you compare SoC to other melee strike abilities, they only have to follow a 2point checklist:

1) Is the ability off of cooldown and/or enough resources in place to activate (ie: rage/energy) - if yes then,

2) Did the yellow hit land or did it Miss, Dodge, Block or Parry?

Whereas he responded with:

Autoattack is not a part of Seal of Command itself, as well as it is not a part of Sinister Strike or any other melee ability. That said, the roll becomes identical:

1) Did the ability go off (you used it, ability procced, it was reactively activated from other source) - if yes, then

2) Did the yellow hit land or not?

**Comment by theloras** [comment]
Rogue Poisons going through Blessing of Protection (both Crippling and Instant):

https://cdn.discordapp.com/attachments/253215035613118467/314650115971350529/rogue_poisons_through_BoP.jpg

**Comment by killerduki** [comment]
@theloras  The Poisons Rogues do is beneficial bug for them , it has nothing with this report.

The thing is , it is working as intended for us , but it is bugged beneficial for Rogues , and Rogues should currently get fix , because their poisons are not meant to go thru BoP .

**Comment by theloras** [comment]
@killerduki so Rogue poisons going through BoP are indeed a bug and not blizzlike?

**Comment by killerduki** [comment]
@theloras  yep Rogue poisons going through BoP while their attacks are immune is a bug and not blizzlike .

If anything goes against that evidence (in case i am wrong), will also prove that SoR should work thru immune swings.

**Comment by theloras** [comment]
ok I figured as much - either the Rogue poisons going through BoP is a bug and not blizzlike, or SoR/SoC should be able to proc through physical invulnerabilities.

it can't be both

**Comment by GuybrushGit** [comment]
Is there any video evidence to go along with this issue?

Only relevant info I found was this:
>If a pally used the blessing of Protection, which mitigates physical damage, but not spells and elemental damage, I would see an "Immune" to my swing, then a very small amount of damage given (I think Iceblade Hacker is 1-4 damage)
>http://www.lurkerlounge.com/forums/thread-4352.html

Meaning even if the hit is immune the weapon still procs. 
We all know how SoR has so many special rules attached to it, but when the evidence is lacking we should go with the default behaviour.
Both seals and poisons should proc. Unless someone has any aditional evidence, I did not look very deep.

**Comment by GuybrushGit** [comment]
Another piece of evidence: 

>All other effects of physical attacks still seem to work through the BoP, including saps, knockdowns, stuns, poisons, etc - all you are preserved from is physical damage. 
>http://wow.allakhazam.com/forum.html?forum=243&mid=1163040089217641225#22

So I'm leaning towards the second option: both poisons and SoR go through the bubble.

**Comment by killerduki** [comment]
Currently SoR does not go through the bubble.

Then you gotta fix it.

**Comment by GuybrushGit** [comment]
Nah I don't feel like it.

**Comment by theloras** [comment]
about that pledge of professionalism...

**Comment by GuybrushGit** [comment]
This is not my profession. I'm not part of the team I'm just a player. WTF are you talking about?

**Comment by theloras** [comment]
@Phoosy - Here you see Xolkar's Seal of Command able to proc off of a white attack that is parried:

_Xolkar 20/0/31 Paladin PvP
Uploaded on Aug 13, 2006_
https://youtu.be/Pk_kHHI0t4o?t=75

**Comment by Phoosy** [comment]
Not sure why you're pinging me on this train of thought - the original reported issue has nothing to do with Parry.

**Comment by theloras** [comment]
@Phoosy you had asked for Vanilla era evidence in one of your earlier posts in this report for Seal of Command being able to proc when the white hit doesn't land:

Phoosy commented 28 days ago
_Evidence from Nost or Feenix is not admissable - only vanilla WoW content is a valid representation._

**Comment by Phoosy** [comment]
I meant in a general sense, not specifically your claim or particular findings. This was back when Nostalrius was regularly being cited for why things were one way while our development team had changed it to another as if that somehow made our changes incorrect or wrong.
